packages/office-ui-fabric-react/src/components/Button/ButtonThemes.ts

```diff
@@ -147,8 +157,8 @@ export function primaryStyles(theme: ITheme): IButtonStyles {
         [`.${IsFocusVisibleClassName} &:focus`]: {
           selectors: {
             ':after': {
-              outline: `none`,
-              borderColor: p.white
+              border: `none`,
+              outlineColor: p.white
```

Comment:
Probably best to specify 
```ts
outline: `1px solid ${p.white}`
```
Because it seems that this change is causing some theme defined outlined to be passed here.